packages/react/src/components/Footer/LocaleButton.js

```diff
@@ -67,27 +33,11 @@ const LocaleButton = ({ selectItem }) => {
         United States — English
```

Comment:
@kennylam as discussed, let's get this as well as the modal heading to be dynamically set based on the html lang attribute.